packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -137,6 +137,11 @@ export default {
         tallBelowMid: {
             type: Boolean,
             default: false
+        },
+
+        useJetLogo: {
```

Comment:
More of a suggestion but would `shouldUseJetLogo` work better as a name?

Reply:
Thanks, changed